reyna/src/com/b2msolutions/reyna/Dispatcher.java

```diff
@@ -32,10 +33,13 @@
 
     private static final String TAG = "com.b2msolutions.reyna.Dispatcher";
 
+    private OutputStreamFactory outputStreamFactory;
+
     protected Clock clock;
 
-    public Dispatcher() {
+    public Dispatcher(OutputStreamFactory outputStreamFactory) {
```

Comment:
can you leave the other constructor in as well? I think we have a few usages of Dispatcher() without args in Elemez and I can't be bothered to update them :D.